.gitignore

```
@@ -14,12 +14,12 @@ Thumbs.db
 
 /Source/Cesium.js
 
-/Source/Shaders/*.js
-/Source/Shaders/*/*.js
-/Source/Shaders/*/*/*.js
-/Source/ThirdParty/Shaders/*.js
-
 /Specs/SpecList.js
+/Source/Shaders/**/*.js
+/Source/ThirdParty/Shaders/**/*.js
+/Source/Workers
```

```suggestion
/Source/Workers/**
!/Source/Workers
```

.gitignore can't include files in a directory that has been excluded completely. So the `!/Source/Workers/cesiumWorkerBootstrapper.js` doesn't actually work without my suggested change above.

---

@kring is correct here, (nice catch).

@OmarShehata what you are seeing is that git has different behavior if you are ignoring a file that is already submitted to the repository.  If you were to create a new file called Workers/foo.js and then tried to unignore it without Kevin's change, it doesn't work.